x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -1918,7 +1910,7 @@ export class RulesClient {
         });
         this.logger.debug(
           `Successfully deleted schedules for underlying tasks: ${taskIdsToDelete
-            .filter((id) => taskIdsFailedToBeDeleted.includes(id))
```

Comment:
Interesting, I am wondering if we should log the other way around. I mean we should log the one that we failed to delete.

Reply:
or maybe both?

Reply:
@XavierM It makes sense. Added this: edbaa488849b19c815f0a17440550191f73e7e9a